[2022-07-22 13:00:32,395] {taskinstance.py:1179} INFO - Dependencies all met for <TaskInstance: tutorial.t2 scheduled__2022-07-11T00:00:00+00:00 [queued]>
[2022-07-22 13:00:32,409] {taskinstance.py:1179} INFO - Dependencies all met for <TaskInstance: tutorial.t2 scheduled__2022-07-11T00:00:00+00:00 [queued]>
[2022-07-22 13:00:32,410] {taskinstance.py:1376} INFO - 
--------------------------------------------------------------------------------
[2022-07-22 13:00:32,410] {taskinstance.py:1377} INFO - Starting attempt 1 of 2
[2022-07-22 13:00:32,411] {taskinstance.py:1378} INFO - 
--------------------------------------------------------------------------------
[2022-07-22 13:00:32,429] {taskinstance.py:1397} INFO - Executing <Task(PythonOperator): t2> on 2022-07-11 00:00:00+00:00
[2022-07-22 13:00:32,434] {standard_task_runner.py:52} INFO - Started process 869 to run task
[2022-07-22 13:00:32,441] {standard_task_runner.py:79} INFO - Running: ['***', 'tasks', 'run', 'tutorial', 't2', 'scheduled__2022-07-11T00:00:00+00:00', '--job-id', '83', '--raw', '--subdir', 'DAGS_FOLDER/nickdag.py', '--cfg-path', '/tmp/tmprryrr3ny', '--error-file', '/tmp/tmp8wrgke37']
[2022-07-22 13:00:32,443] {standard_task_runner.py:80} INFO - Job 83: Subtask t2
[2022-07-22 13:00:32,475] {logging_mixin.py:115} WARNING - /home/***/.local/lib/python3.7/site-packages/***/configuration.py:528 DeprecationWarning: The sql_alchemy_conn option in [core] has been moved to the sql_alchemy_conn option in [database] - the old setting has been used, but please update your config.
[2022-07-22 13:00:32,569] {task_command.py:371} INFO - Running <TaskInstance: tutorial.t2 scheduled__2022-07-11T00:00:00+00:00 [running]> on host 2f63307804d5
[2022-07-22 13:00:32,696] {taskinstance.py:1591} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_EMAIL=caroline.kosalka@gmail.com
AIRFLOW_CTX_DAG_OWNER=***
AIRFLOW_CTX_DAG_ID=tutorial
AIRFLOW_CTX_TASK_ID=t2
AIRFLOW_CTX_EXECUTION_DATE=2022-07-11T00:00:00+00:00
AIRFLOW_CTX_TRY_NUMBER=1
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2022-07-11T00:00:00+00:00
[2022-07-22 13:00:32,697] {python.py:173} INFO - Done. Returned value was: goodbye
[2022-07-22 13:00:32,745] {taskinstance.py:1420} INFO - Marking task as SUCCESS. dag_id=tutorial, task_id=t2, execution_date=20220711T000000, start_date=20220722T130032, end_date=20220722T130032
[2022-07-22 13:00:32,814] {local_task_job.py:156} INFO - Task exited with return code 0
[2022-07-22 13:00:32,890] {local_task_job.py:273} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2022-07-25 09:18:15,403] {taskinstance.py:1179} INFO - Dependencies all met for <TaskInstance: tutorial.t2 scheduled__2022-07-11T00:00:00+00:00 [queued]>
[2022-07-25 09:18:15,418] {taskinstance.py:1179} INFO - Dependencies all met for <TaskInstance: tutorial.t2 scheduled__2022-07-11T00:00:00+00:00 [queued]>
[2022-07-25 09:18:15,419] {taskinstance.py:1376} INFO - 
--------------------------------------------------------------------------------
[2022-07-25 09:18:15,419] {taskinstance.py:1377} INFO - Starting attempt 1 of 2
[2022-07-25 09:18:15,420] {taskinstance.py:1378} INFO - 
--------------------------------------------------------------------------------
[2022-07-25 09:18:15,440] {taskinstance.py:1397} INFO - Executing <Task(PythonOperator): t2> on 2022-07-11 00:00:00+00:00
[2022-07-25 09:18:15,445] {standard_task_runner.py:52} INFO - Started process 835 to run task
[2022-07-25 09:18:15,452] {standard_task_runner.py:79} INFO - Running: ['***', 'tasks', 'run', 'tutorial', 't2', 'scheduled__2022-07-11T00:00:00+00:00', '--job-id', '85', '--raw', '--subdir', 'DAGS_FOLDER/nickdag.py', '--cfg-path', '/tmp/tmpg5lf3rge', '--error-file', '/tmp/tmpx3zw6h17']
[2022-07-25 09:18:15,454] {standard_task_runner.py:80} INFO - Job 85: Subtask t2
[2022-07-25 09:18:15,484] {logging_mixin.py:115} WARNING - /home/***/.local/lib/python3.7/site-packages/***/configuration.py:528 DeprecationWarning: The sql_alchemy_conn option in [core] has been moved to the sql_alchemy_conn option in [database] - the old setting has been used, but please update your config.
[2022-07-25 09:18:15,572] {task_command.py:371} INFO - Running <TaskInstance: tutorial.t2 scheduled__2022-07-11T00:00:00+00:00 [running]> on host 01e6b1df1873
[2022-07-25 09:18:15,676] {taskinstance.py:1591} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_EMAIL=caroline.kosalka@gmail.com
AIRFLOW_CTX_DAG_OWNER=***
AIRFLOW_CTX_DAG_ID=tutorial
AIRFLOW_CTX_TASK_ID=t2
AIRFLOW_CTX_EXECUTION_DATE=2022-07-11T00:00:00+00:00
AIRFLOW_CTX_TRY_NUMBER=1
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2022-07-11T00:00:00+00:00
[2022-07-25 09:18:15,678] {python.py:173} INFO - Done. Returned value was: goodbye
[2022-07-25 09:18:15,719] {taskinstance.py:1420} INFO - Marking task as SUCCESS. dag_id=tutorial, task_id=t2, execution_date=20220711T000000, start_date=20220725T091815, end_date=20220725T091815
[2022-07-25 09:18:15,785] {local_task_job.py:156} INFO - Task exited with return code 0
[2022-07-25 09:18:15,838] {local_task_job.py:273} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2022-07-25 12:28:28,299] {taskinstance.py:1179} INFO - Dependencies all met for <TaskInstance: tutorial.t2 scheduled__2022-07-11T00:00:00+00:00 [queued]>
[2022-07-25 12:28:28,387] {taskinstance.py:1179} INFO - Dependencies all met for <TaskInstance: tutorial.t2 scheduled__2022-07-11T00:00:00+00:00 [queued]>
[2022-07-25 12:28:28,390] {taskinstance.py:1376} INFO - 
--------------------------------------------------------------------------------
[2022-07-25 12:28:28,391] {taskinstance.py:1377} INFO - Starting attempt 1 of 2
[2022-07-25 12:28:28,392] {taskinstance.py:1378} INFO - 
--------------------------------------------------------------------------------
[2022-07-25 12:28:28,482] {taskinstance.py:1397} INFO - Executing <Task(PythonOperator): t2> on 2022-07-11 00:00:00+00:00
[2022-07-25 12:28:28,491] {standard_task_runner.py:52} INFO - Started process 1030 to run task
[2022-07-25 12:28:28,542] {standard_task_runner.py:79} INFO - Running: ['***', 'tasks', 'run', 'tutorial', 't2', 'scheduled__2022-07-11T00:00:00+00:00', '--job-id', '169', '--raw', '--subdir', 'DAGS_FOLDER/nickdag.py', '--cfg-path', '/tmp/tmpovn6db8i', '--error-file', '/tmp/tmpzkarxq4r']
[2022-07-25 12:28:28,544] {standard_task_runner.py:80} INFO - Job 169: Subtask t2
[2022-07-25 12:28:28,676] {logging_mixin.py:115} WARNING - /home/***/.local/lib/python3.7/site-packages/***/configuration.py:528 DeprecationWarning: The sql_alchemy_conn option in [core] has been moved to the sql_alchemy_conn option in [database] - the old setting has been used, but please update your config.
[2022-07-25 12:28:29,100] {task_command.py:371} INFO - Running <TaskInstance: tutorial.t2 scheduled__2022-07-11T00:00:00+00:00 [running]> on host 929e6a3728d4
[2022-07-25 12:28:29,589] {taskinstance.py:1591} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_EMAIL=caroline.kosalka@gmail.com
AIRFLOW_CTX_DAG_OWNER=***
AIRFLOW_CTX_DAG_ID=tutorial
AIRFLOW_CTX_TASK_ID=t2
AIRFLOW_CTX_EXECUTION_DATE=2022-07-11T00:00:00+00:00
AIRFLOW_CTX_TRY_NUMBER=1
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2022-07-11T00:00:00+00:00
[2022-07-25 12:28:29,592] {python.py:173} INFO - Done. Returned value was: goodbye
[2022-07-25 12:28:29,920] {taskinstance.py:1420} INFO - Marking task as SUCCESS. dag_id=tutorial, task_id=t2, execution_date=20220711T000000, start_date=20220725T122828, end_date=20220725T122829
[2022-07-25 12:28:30,101] {local_task_job.py:156} INFO - Task exited with return code 0
[2022-07-25 12:28:30,318] {local_task_job.py:273} INFO - 0 downstream tasks scheduled from follow-on schedule check
